### DISCUSSION TITLE
Is it possible to use authelia to secure a forward proxy?

### DISCUSSION THREAD

**Discussion by christaikobo** [authelia/authelia#7999] [Q&A]
As the title suggests, I have a forward proxy at `https://proxy.example.com`, I would like to add a basic auth to it using authelia.

But going through all the documents, I don't seem to find a way to do it other than changing default policy to `one_factor` which is less than ideal, because there is no ACL that can catch a forward proxy request.

I have being banging my head over this for a couple of hours. Maybe there is a smart guy out here who have a better solution to this?

**Comment by james-d-elliott** [comment]
It's not exactly something we've intended use cases for, but it would theoretically be possible using the networks option and ensuring the proxy is the IP of the request.

**Reply by christaikobo** [reply]
Thanks for the suggestion, however I'm a bit confused. The network ACL control applies to source IP, so are you suggesting to put authelia **after** the forward proxy? Is this even possible?

**Reply by christaikobo** [reply]
It's fine, I understand authelia is not designed to do this. You have pointed out a possibility, and I will see whether it is possible. Maybe it's not, and that's fine.
